### PR TITLE
Added dodge Queer Bubble

### PR DESCRIPTION
**Pull request by domesticwarlord86**

### CODE REVIEW

File: Plugins/Trust/Dungeons/BrayfloxsLongstop.cs [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+        /// Gets zone ID for this dungeon.
+        /// </summary>
+        public new const ZoneId ZoneId = Data.ZoneId.BrayfloxsLongstop;
+
+        private const int GreatYellowPelican = 1280;
+        private const int InfernoDrake = 1284;
+        private const int Hellbender = 1286;
+        private const int bubbleObj = 1383;
+        private const int Aiatar = 1279;
+
```
Comment (TheManta) [inline Plugins/Trust/Dungeons/BrayfloxsLongstop.cs:25]: Code should not contain consecutive empty lines, per Style Cop ruleset.

File: Plugins/Trust/Dungeons/BrayfloxsLongstop.cs [hunk 2]
```diff
@@ -1,25 +1,48 @@
...
+        /// <inheritdoc/>
+        public override DungeonId DungeonId => DungeonId.BrayfloxsLongstop;
+
+        /// <inheritdoc/>
+        public override async Task<bool> RunAsync()
+        {
+            BattleCharacter hellbenderNpc = GameObjectManager.GetObjectsByNPCId<BattleCharacter>(NpcId: Hellbender)
+                .FirstOrDefault(bc => bc.Distance() < 50 && bc.IsTargetable);
+            if (hellbenderNpc != null && hellbenderNpc.IsValid)
```
Comment (TheManta) [inline Plugins/Trust/Dungeons/BrayfloxsLongstop.cs:35]: Unless `hellbenderNpc` will be used in a future iteration, remove the `GetObjectsByNPCId()` and change this condition to check for boss arena sub-zone and if we're in combat.

As-is, this spends time every tick throughout the dungeon searching for an NPC to be allowed to search for the NPC we care about. Removing the unnecessary Hellbender search and limiting it to searching the Queer Bubble only when in the right place will improve performance for the whole dungeon run.

**Review by TheManta — CHANGES_REQUESTED** [review]
I wanted to practice GitHub code reviews, so here we are.

Most issues can be fixed in Rider by going to `Problems (Alt-8) > Current File` and reading the ⚠️ items shown. "Typos" in monster names and "async method doesn't await" error on `RunAsync()` can be ignored. Leaving unused boss name => NpcId constants is also fine.

image

File: Plugins/Trust/Dungeons/BrayfloxsLongstop.cs [hunk 3]
```diff
@@ -1,25 +1,48 @@
...
+    {
+        /// <summary>
+        /// Gets zone ID for this dungeon.
+        /// </summary>
+        public new const ZoneId ZoneId = Data.ZoneId.BrayfloxsLongstop;
+
+        private const int GreatYellowPelican = 1280;
+        private const int InfernoDrake = 1284;
+        private const int Hellbender = 1286;
+        private const int bubbleObj = 1383;
```
[the remark this answers is not in the document]
Comment (domesticwarlord86) [reply Plugins/Trust/Dungeons/BrayfloxsLongstop.cs]: Fixed

File: Plugins/Trust/Dungeons/BrayfloxsLongstop.cs (hunk 1 above)
Comment (domesticwarlord86) [reply Plugins/Trust/Dungeons/BrayfloxsLongstop.cs:25]: fixed

File: Plugins/Trust/Dungeons/BrayfloxsLongstop.cs (hunk 2 above)
Comment (domesticwarlord86) [reply Plugins/Trust/Dungeons/BrayfloxsLongstop.cs]: fixed

**Comment by domesticwarlord86** [comment]
Should be fixed

**Review by TheManta — APPROVED** [review]

**Pull request merged**